packages/cozy-client/src/models/document/documentTypeData.js

```diff
@@ -170,6 +171,23 @@ const invoiceLabels = [
   'other_invoice'
 ]
 
+/**
+ * @type {Array<import("../../types").OthersLabels>}
+ */
+export const othersLabels = ['other_administrative_document']
+
+/**
+ * @type {import("../../types").Theme}
+ */
+const healthTheme = flag('hide.healthTheme.enabled')
+  ? undefined
+  : {
+      id: 'theme4',
+      label: 'health',
+      icon: 'heart',
+      items: buildItemsByLabel(healthLabels)
+    }
+
```

Comment:
I feel like the `healthTheme` should always be here, and it's the calling app that should use the flag to decide if it should be included or not. Otherwise, that sounds like are we putting some business logic in a shared lib

Reply:
C'était la question, on voulait éviter de filtrer sur toutes les apps, mais si ça te semble plus pertinent comme ça je peux changer 👍